extensions/ftp/src/main/java/org/mule/extension/ftp/internal/FtpConnector.java

```diff
@@ -33,6 +34,7 @@
 @Operations(StandardFileSystemOperations.class)
 @Providers({FtpConnectionProvider.class})
 @Xml(schemaLocation = "http://www.mulesoft.org/schema/mule/ftp", namespace = "ftp", schemaVersion = "4.0")
+@Studio(provided = true)
```

Comment:
shouldn't we have studio generate the editor? At least until the studio team builds custom editors that should be the case I think